salt/proxy/init.sls

```diff
@@ -1,18 +1,25 @@
 {% if salt['pillar.get']('proxy:systemwide', '').lower() == 'true' %}
 
+{% set proxy_http  = salt['pillar.get']('proxy:http', '') %}
+{% set proxy_https = salt['pillar.get']('proxy:https', '') %}
+{% set no_proxy    = salt['pillar.get']('proxy:no_proxy', '') %}
+
 /etc/sysconfig/proxy:
   file.managed:
     - makedirs: True
     - contents: |
         PROXY_ENABLED="yes"
```

Comment:
I think it's harmless, as we've checkd for the proxy:systemwide pillar outside, and that means we have some sort of proxy to be enabled... If thats enabled, and no proxy has been supplied, it's a end user configuration error anyway IMO